### PR TITLE
TJJ-001: Created Delivery Employee Table

### PR DESCRIPTION
**Pull request by dylanmckk**

### CODE REVIEW

**Review by MatthewJKainos — CHANGES_REQUESTED** [review]
Indentation has been done using spaces.
Could we update these to use tabbing?

**Review by MatthewJKainos — APPROVED** [review]

File: src/main/resources/db/migration/V2__create_deliv_emp_tbl.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE DeliveryEmployee (
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V2__create_deliv_emp_tbl.sql:1]: dont like this file name

File: src/main/resources/db/migration/V2__create_deliv_emp_tbl.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+	id int NOT NULL,
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V2__create_deliv_emp_tbl.sql:2]: what is id?

**Review by lukemKainos — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V2__create_deliv_emp_tbl.sql (hunk 1 above)
Comment (dylanmckk) [reply src/main/resources/db/migration/V2__create_deliv_emp_tbl.sql:1]: https://mtlynch.io/human-code-reviews-1/

**Review by lukemKainos — APPROVED** [review]

**Pull request merged**